apps/snitch_core/test/data/model/stock/stock_movement_test.exs

```diff
@@ -0,0 +1,75 @@
+defmodule Snitch.Data.Model.StockMovementTest do
+  @moduledoc false
+
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+  import Snitch.Factory
+  alias Snitch.Data.Model.StockMovement, as: StockMovementModel
+
+  setup do
+    [stock_item: insert(:stock_item)]
+  end
+
+  describe "create/4" do
+    test "Fails for INVALID stock item id" do
+      assert {:error, changeset} = StockMovementModel.create(1, -1)
+      refute changeset.valid?
+      assert %{stock_item_id: ["does not exist"]} = errors_on(changeset)
+    end
+
+    test "Fails with invalid quantity", context do
```

Comment:
No need to test the capability of `cast`. Don't slow down our test suite, it's getting big very fast!

Reply:
Let's say someone changes the error message for quantity, without testing it. Then this could catch it.

Reply:
How would someone change the error message created by `cast`, `cast` takes no such argument?